docs/api-reference/core/view.md

```diff
@@ -81,18 +81,37 @@ The `viewState` property is intended to support a number of use cases:
 * Overriding a partial set of view state properties from a selected view state.
 
 
-#### `clear` (boolean | object, optional) {#clear}
+#### `clear` (boolean, optional) {#clear}
 
-Clears the contents (pixels) of the viewport. The value of the `clear` prop is passed as an argument to luma.gl's `clear` function. If `true` clears color and depth buffers. If an object, behaviour is controlled by the following fields:
+Clears the contents (pixels) of the viewport. If `true` clears color, depth, and stencil buffers. Behavior is controlled with the `clearColor`, `clearDepth`, and `clearStencil` properties.
 
-* `color` (boolean or Array) - if not `false`, clears all active color buffers with either the provided color or the currently set clear color.
-* `depth` (boolean)  - if `true`, clears the depth buffer.
-* `stencil` (boolean) - if `true` clears the stencil buffer.
-
-Note that deck.gl always clears the screen before each render, and clearing, while cheap, is not totally free. This means that viewports should only specify the `clear` property if they need additional clearing, e.g. because they are rendering on top of another viewport, or want to have a different background color etc.
+Note that deck.gl always clears the screen before each render, and clearing, while cheap, is not totally free. This means that viewports should only clear the viewport if they need additional clearing, e.g. because they are rendering on top of another viewport, or want to have a different background color etc.
 
 Default `false`.
 
+#### `clearColor` (number[] | false, optional) {#clearcolor}
+
+Specifies the color to clear the viewport with, as an array of four numbers `[r, g, b, a?]`. Each channel should be an integer between 0 and 255. For example, `[255, 0, 0, 255]` for opaque red. If `clearColor` is `false`, the depth buffer will not be cleared. If `clear` is set to `false`, this property will be ignored.
```

Comment:
Just thinking to myself, doubt that we'll get to any meaningful concluson at this point.
- A little confusing that this almost copies luma.gl props but "quitely" changes the range of one of them.
- Maybe we could offer a separate prop to distinguish color scales, like `clearColor255`